### DISCUSSION TITLE
Compilation error for ActionArgs even if i'm using the right remix version

### DISCUSSION THREAD

**Discussion by Bistaff** [remix-run/remix#7600] [Q&A]
Im' using the example from this page [(Form vs Fetcher),](https://remix.run/docs/en/main/discussion/form-vs-fetcher) and I have errors on imports.
How can It be possibile if i'm using the same version?

`Module  "@remix-run/node"  has no exported member  ActionArgs `

**Comment by dmarkow** [comment]
Looks like the documentation needs an update, `LoaderArgs` and `ActionArgs` were renamed to `LoaderFunctionArgs` and `ActionFunctionArgs` in v2.

https://github.com/remix-run/remix/releases/tag/remix%402.0.0

**Reply by kiliman** [reply]
At this point, I would recommend using `DataFunctionArgs` in both places, as the others are just aliases.

**Comment by agarcher** [comment]
Same issue on https://remix.run/docs/en/main/guides/form-validation

Edit: I opened https://github.com/remix-run/remix/pull/7718 to address this and one other issue in the form validation guide
